mod_reforged/hooks/items/accessory/accessory.nut

```diff
@@ -0,0 +1,30 @@
+::mods_hookExactClass("items/accessory/accessory", function(o) {
```

Comment:
Please drop this commit. It should already have been merged into development

It somehow sneaked into your PR but has nothing to do with it.

Reply:
fixed